app/database/result_store_propagate.go

```diff
@@ -28,7 +29,8 @@ const (
 //  5. We process all objects that are marked as 'to_be_recomputed' and that have no children marked as 'to_be_recomputed'.
 //     Then, if an object has children, we update
 //     latest_activity_at, tasks_tried, tasks_with_help, validated_at.
-//  6. We mark all results marked as 'to_be_recomputed' as 'to_be_propagated'.
+//  6. We mark all modified results marked as 'to_be_recomputed' as 'to_be_propagated' and
+//     unmark all unchanged results marked as 'to_be_recomputed'.
```

Comment:
shouldn't you "text workflow" mention this new chunk size ?

Reply:
fixed